src/main/java/org/rumbledb/expressions/typing/TreatExpression.java

```diff
@@ -40,7 +40,7 @@ public SequenceType getsequenceType() {
     }
 
     public boolean shouldThrowTreatException() {
-        return shouldThrowTreatException;
+        return this.shouldThrowTreatException;
     }
 
     @Override
```

Comment:
Can we make the calculateIsRDDFromSequenceTypeAndExpression method of this class private now?

Reply:
DONE